lib/menu.rb

```diff
@@ -0,0 +1,24 @@
+require_relative './dish.rb'
+require_relative './order.rb'
+
+class Menu
+  attr_reader :menu_list
+
+  def initialize
+    @dishes = []
+  end
+
+  def add_dish(dish)
+    @dishes << dish
+  end
+
+  def show_menu
+    @dishes.each do |dish|
+      puts "#{dish.name} - #{dish.price}"
+    end
+  end
+
+  def create_order(*dish)
+    Order.new(*dish)
+  end
+end
```

Comment:
Nice, this makes sense. It looks like it adds any dishes in the argument to a new instance of the Order class